packages/components/src/DescriptionList/DescriptionList.test.tsx

```diff
@@ -14,6 +14,21 @@ it("renders an object as a list of key value pairs", () => {
   expect(container).toMatchSnapshot();
 });
 
+it("allows duplicate terms", () => {
```

Comment:
if this is testing both allowing and not throwing the key error, i'd like that to be reflected here